package test.associate.exceptions;

public class InvalidCheckedExceptionDemo1 {
    static Integer i;

    public static void main(String[] args) {
        print();
    }

    public static void print() {
        /*try {
            System.out.println(i++);
        } catch (IOException ex) {
            System.out.println("error");
        }*/
    }
}
